### PR TITLE
Travel Reimbursement

### PR DESCRIPTION
**Pull request by chiayi**
It now gives reimbursement base on the table.
Had to use random generator for testing because don't have google API key.
I added a dictionary to the config file for matching the car rate, how do I add that to the main config file?

### CODE REVIEW

**Comment by chiayi** [comment]
Its not passing the check because the dict added to my config does not exist in the tested config.

File: reimburse.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from pymongo import UpdateOne
 from pymongo.errors import BulkWriteError
 
+import random
```
Comment (mjrb) [inline reimburse.py]: nit: we can remove this if it was just for testing

**Comment by mjrb** [comment]
oh, also (not pictured and unrelated) can you update the qr_link endpoint so volunteers can use it too?

**Comment by mjrb** [comment]
oops, i just added the api key to travis and re ran the build

**Comment by mjrb** [comment]
lmao, ye olde linter error. see build log

**Comment by mjrb** [comment]
Also can you remove the dayof option and just have it do this lookup on any user who isn't "unregistered"

File: reimburse.py [hunk 2]
```diff
@@ -61,22 +61,21 @@ def req_distance_matrices(users):
...
+def users_to_reimburse(lookup, users):
     total = 0
     table = dict()
     for user in users:
-        if user['travelling_from']['mode'] != 'plane':
+        if user['travelling_from']['mode'] == 'car':
             dist = lookup[user['travelling_from']['mode']].get(user['travelling_from']['formatted_addr'], 0)
-            reimburse = min(dist * config.TRAVEL.MULTIPLIERS[user['travelling_from']['mode']], config.TRAVEL.MAX_REIMBURSE)
+            dist = round(2*dist/1609.344)
+            result = [config.TRAVEL.CAR_RATE[key] for key in config.TRAVEL.CAR_RATE if dist in key]
```
Comment (mjrb) [inline reimburse.py]: interesting way of doing this :+1:

File: tests/test_reimburse.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        ]
+
+
+def payload(auth_email=email):
+    return {"email" : email,
+            "token" : token}
+
+def setup_module(m):
+    db = util.get_db()
+    db["test-users"].drop()
```
Comment (mjrb) [inline tests/test_reimburse.py]: maybe we should make a ticket for decorators for common setup/teardown stuff. like making a user or switching to a test database

**Pull request merged**